### PR TITLE
Bugfix/too many py42s

### PR DESCRIPTION
**Pull request by unparalleled-js**
*Fixes issue in de bulk add where we would instantiate a py42 per row in the csv file.

### CODE REVIEW

File: src/code42cli/cmds/departing_employee.py [hunk 1]
```diff
@@ -102,26 +100,21 @@ def bulk(state):
 )
 @read_csv_arg(headers=DEPARTING_EMPLOYEE_CSV_HEADERS)
 @sdk_options()
-@click.pass_context
-def bulk_add(ctx, state, csv_rows):
+def bulk_add(state, csv_rows):
+    sdk = state.sdk  # Force initialization of py42 to only happen once.
```
Comment (unparalleled-js) [inline src/code42cli/cmds/departing_employee.py:104]: it seems as though all the other bulk commands already are doing this

File: src/code42cli/cmds/departing_employee.py [hunk 2]
```diff
@@ -102,26 +100,21 @@ def bulk(state):
...
                 )
                 raise Code42CLIError(message)
-        ctx.invoke(
-            add,
-            username=username,
-            cloud_alias=cloud_alias,
-            departure_date=departure_date,
-            notes=notes,
-        )
+        _add_departing_employee(sdk, username, cloud_alias, departure_date, notes)
```
Comment (unparalleled-js) [inline src/code42cli/cmds/departing_employee.py:117]: `ctx.invoke()` was the culprit here but I still like this better. I don't know what `ctx.invoke()` does behind the scenes and it seems like unnecessary to handle off the execution back to the context

**Comment by unparalleled-js** [comment]
Should I include a changelog like `Fix performance issues with code42 departing-employee bulk add` or something?

**Review by alanag13 — APPROVED** [review]

**Comment by alanag13** [comment]
> Should I include a changelog like `Fix performance issues with code42 departing-employee bulk add` or something?

Sure, that wouldn't hurt.

**Pull request merged**